### DISCUSSION TITLE
Faster transcription using JAX

### DISCUSSION THREAD

**Discussion by Mage-Enderman** [chidiwilliams/buzz#560] [Ideas]
https://github.com/sanchit-gandhi/whisper-jax

Found out about it from this video
https://www.youtube.com/watch?v=E6-V86n61Qg

**Comment by walkersevenonethreenineone** [comment]
Is it possible to install this JAX Whisper for offline usage?
I just can't find any solution...
Especially for Windows...

**Comment by Lucas7AC** [comment]
How do I put this on buzz?

**Comment by raivisdejus** [comment]
My testing of `whisper-jax` did not show significantly faster results than `faster-whisper` on CPU or GPU. The claim of 70x faster speed is on specialized TPU hardware that most people usually do not have.

`whisper-jax` is currently not available in  buzz

**Comment by raivisdejus** [comment]
A few notes for future:
- JAX needs a separate version of jaxlib for CPU and GPU, to implement it in Buzz this should be sorted out.
- Batched processing for Faster whisper is very fast, as claimed on their Github 3-4x faster than regular faster  whisper. Once that has stable release, it could be implemented in Buzz.
